src/managers/GuildEmojiRoleManager.js

```diff
@@ -71,7 +71,10 @@ class GuildEmojiRoleManager {
   remove(roleOrRoles) {
     if (roleOrRoles instanceof Collection) return this.remove(roleOrRoles.keyArray());
     if (!Array.isArray(roleOrRoles)) return this.remove([roleOrRoles]);
-    roleOrRoles = roleOrRoles.map(r => this.guild.roles.resolveID(r));
+    roleOrRoles = roleOrRoles.map(role => {
+      const resolvedRole = this.guild.roles.resolve(role);
+      return resolvedRole && resolvedRole.id;
```

Comment:
```suggestion
      return resolvedRole?.id;
```

Reply:
that would return `undefined | string`, `null | string` is the wanted return type here

Reply:
Then slap a `?? null` on it :D

Reply:
I think it can reasonably stay like it is now (changed).